neaten/neaten.go

```diff
@@ -147,8 +136,8 @@ func (t *Tidy) moveAndDelete() error {
 // ownership and permissions to match the destDir.
 func (t *Tidy) findAndMoveOutputs(inSuffix, outSuffix string) error {
 	outputPaths, err := filepath.Glob(fmt.Sprintf(t.CombineFileGlobPattern, t.SrcDir, inSuffix))
-	if err != nil {
-		return err
+	if err != nil || len(outputPaths) == 0 {
+		panic(fmt.Sprintf("could not find %s files in %s", inSuffix, t.SrcDir))
```

Comment:
Why panic? Just return the error if != nil. If no paths, return a new Error type with that message.